### PR TITLE
add as923_1b region

### PR DESCRIPTION
**Pull request by mikev**
AS923_1B defines a Region supporting Malaysia. It is illegal in Malaysia to use any frequency above 924 Mhz, hence the reason for this new Region. The AS923_1B defines Uplink frequencies at [922.0, 922.2, 922.4, 922.6, 922.8, 923.0, 923.2, 923.4]

I don't believe this change broke any CT tests, but some CT tests still fail on buildkite and some tests were commented to make further test progress on buildkite.

This change is a prerequisite for - https://github.com/helium/miner/pull/1664

### CODE REVIEW

File: test/blockchain_region_SUITE.erl [hunk 1]
```diff
@@ -213,6 +217,14 @@ as923_1_test(Config) ->
     false = blockchain_region_v1:h3_in_region(H3, region_us915, Ledger),
     ok.
 
+as923_1b_test(Config) ->
+    Ledger = ?config(ledger, Config),
+    H3 = 631319855840474623,
```
Comment (vihu) [inline test/blockchain_region_SUITE.erl:222]: Maybe add a comment here stating where this H3 index is located?

**Comment by vihu** [comment]
I think this looks good generally. We need to start consolidating all the necessary chain vars. The ones with `region_*` are the most cumbersome ones as they depend on serialized regions from <https://github.com/helium/lorawan-h3>

File: test/blockchain_region_SUITE.erl (hunk 1 above)
Comment (mikev) [reply test/blockchain_region_SUITE.erl]: Add comment with link to H3 and HPlans repos

**Review by Vagabond — APPROVED** [review]
All this adds is symbol lookups for the new region, everything else is no-op or testing related.

**Review by vihu — APPROVED** [review]

**Pull request merged**